lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -144,7 +148,7 @@ Ufe::AppendedChild UsdHierarchy::appendChild(const Ufe::SceneItem::Ptr& child)
     auto           stage = childPrim.GetStage();
     auto           ufeSrcPath = usdChild->path();
     auto           usdSrcPath = childPrim.GetPath();
-    auto           ufeDstPath = fItem->path() + childName;
+    auto           ufeDstPath = stripInstanceIndexFromUfePath(path()) + childName;
```

Comment:
Again, not sure from a data model perspective why we need the strip call.

Reply:
I think you're right here too, though I'm admittedly not sure about the proper way to handle this when point instances and `PointInstancer` prims are either the parent or the child. This function is also UFE v1 only, so I'm not sure if it would still come into play?

Reply:
Reverted these changes as part of 95a4d4174531e488facd72c1789ebef3434ff469, though I'm not certain whether we need some kind of handling for append/insert when `PointInstancer` and/or point instances are involved as either the parent or the child?